Update Triage work and integrate most of the wiki content (reduces redundancy) (#42)
canonical/ubuntu-maintainers-handbook: BugTriage.md

    leave the main task *open* in this case because otherwise Launchpad
    may not display it in reports and lists.

* Already Fixed in Debian.  
  If the issue has been solved in Debian, it
  will likely be worth merging and/or sru'ing the fix.
  - Make sure steps to reproduce it are identified.

>   - If the issue affects the development release, it is a merge

Author: The details of that board/LP approach are for a different file and different PR. But indeed it needs to talk about "Launchpad bugs" for merges nowadays.

Also added details about "raising the issue" as you requested

    determine which supported Ubuntu releases will need the fix and add
    Bug Tasks as appropriate.

* Already Fixed Upstream.  
  All the steps of the "already fixed in Debian" category above apply,
  here as well - in addition when appropriate ensure to help Debian as

>   well by:

Reviewer: Less wordy to say "- in addition, help Debian when appropriate by:"

Author: Thanks - updated

  - filing a Debian bug about it or chime in if there is an existing one

>   - If your create a PR for Ubuntu that can be uses almost as-is consider

Reviewer: s/your/you/
s/uses/used/

        * Since mysql-8.0 we no more use that tag, instead it turned out to be more reliable to just look at recent
          [mysql-8.0 bugs by heat](https://bugs.launchpad.net/ubuntu/+source/mysql-8.0/+bugs?orderby=-heat&start=0) to spot the
          duplication candidates
    * If not a duplicate, then still please update the bug title from the usual apport "failed on postinst" to whatever the bug
      really is about for better recognizing the issue in any kind of overview that just lists the title.

* Virtualization controlled through libvirt  
  If these reports are about the inability to access devices or *permission denied* issues the user often does not realize that
  libvirt applies an apparmor profile to the guest for enhanced security. If not available in the bug report (dmesg of the time
  of occurance) please make sure to ask to check for apparmor denials at the time the problem triggers.

## Process and Policy

### Direct team subscriptions

We subscribe *~ubuntu-server* directly to a bug to track our community bug backlog when the bug meets the following criteria.
When the bug no longer meets these criteria, we unsubscribe it:

> 1. Anything that, if the bug turns out to valid, is something that would be under the *~ubuntu-server* remit to fix (common use

Reviewer 2: ```suggestion
1. Anything that, if the bug turns out to be valid, is something that would be under the *~ubuntu-server* remit to fix (common use
```

Author: Thanks - updated

   cases but not obscure ones - but nothing stops an individual volunteering to work on an obscure use case, of course).
2. By definition, if it's something that we wouldn't fix and request volunteers even if we had time, then it doesn't warrant a
   subscription.
3. This subscription is for the Ubuntu Server triage community and is not for tracking of internal Canonical customer requests.   
   Whether a Canonical customer has made a request in relation to a particular bug makes no difference and provides no
   additional priority under this process. A Canonical customer bug may still be subscribed if it qualifies under these
   criteria.

> 4. If the bug is assigned to someone on our team, leave it subscribed. No need to subscribe, and feel free to unsubscribe.

Reviewer 2: Could we say "and feel free to unsubscribe the team." here just to be even more explicit?

Author: Thanks - updated

pings/mentions/CC.

## Triage Rotation

According to load we might shift things, but generally every day Tue-Fri
has a team member assigned. Monday is often more work and includes more
low quality bugs as it includes all of the weekend - therefore Monday is
a rotation through all eligible bug triagers.

> This is organized internally int he teams Jira and automation will create

Reviewer 2: ```suggestion
This is organized internally in the team's Jira and automation will create
```

Author: Thanks - updated
